droidlet/dashboard/web/src/components/Timeline/TimelineDropdown.js

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright (c) Facebook, Inc. and its affiliates.
+*/
+
+import React from "react";
+import { handleSearch } from "./TimelineUtils";
+import { makeStyles } from "@material-ui/core/styles";
+import Input from "@material-ui/core/Input";
+import InputLabel from "@material-ui/core/InputLabel";
+import MenuItem from "@material-ui/core/MenuItem";
+import FormControl from "@material-ui/core/FormControl";
+import ListItemText from "@material-ui/core/ListItemText";
+import Select from "@material-ui/core/Select";
+import Checkbox from "@material-ui/core/Checkbox";
+
+const useStyles = makeStyles((theme) => ({
+  formControl: {
+    margin: theme.spacing(1),
+    minWidth: 120,
+    maxWidth: 300,
+  },
+}));
+
+const ITEM_HEIGHT = 48;
+const ITEM_PADDING_TOP = 8;
+const MenuProps = {
+  PaperProps: {
+    style: {
+      maxHeight: ITEM_HEIGHT * 4.5 + ITEM_PADDING_TOP,
+      width: 250,
+    },
+  },
+};
+
+const names = ["Perceive", "Dialogue", "Interpreter", "Memory"];
```

Comment:
The names of four components (["Perceive", "Dialogue", "Interpreter", "Memory"]) appear multiple times in the code. Try to define it once and import/reuse it.